tests/test_api.py

```diff
@@ -187,19 +130,33 @@ def test_get_variant_hashes_by_priority_roundtrip(configs):
     )
 )
 def test_get_variant_hashes_by_priority_roundtrip_fuzz(mocker, configs):
-    def filter_long_combinations():
+    namespace_priorities = list({provider_cfg.namespace for provider_cfg in configs})
+    mocker.patch(
+        "variantlib.configuration.VariantConfiguration.get_config"
+    ).return_value = VConfigurationModel(namespace_priorities=namespace_priorities)
+
+    def get_or_skip_combinations() -> Generator[VariantDescription]:
         for i, x in enumerate(get_combinations(configs)):
-            assume(i < 65536)
+            if i >= 65536:
+                pytest.skip("Too many combinations")
```

Comment:
Why are you changing this? `assume()` helps hypothesis choose better samples, `skip()` just kills the test entirely.

Reply:
It's apparently deprecated. I got a warning message. Replace it back with assume you'll see the message

Reply:
No, it's not. I'm guessing you got a warning because you added it to `test_get_variant_hashes_by_priority_roundtrip()` which doesn't use Hypothesis, and it makes sense only when you are actually using Hypothesis.

Reply:
Ahhh that would explain. My bad I'll revert :)